test_data/scripts/run_domains.py

```diff
@@ -1,60 +0,0 @@
-import os
```

Comment:
probably should be a rename instead of a delete so it does not show up in the repo as a new file.